package.json

```diff
@@ -33,6 +33,7 @@
   "scripts": {
     "bootstrap": "pnpm install",
     "build": "turbo run build",
+    "build:packages": "turbo run build --filter @kuma-ui/*",
```

Comment:
Add faster build script since we don't need to build examples and website for most tasks.